packit_service/worker/handlers/github_handlers.py

```diff
@@ -40,6 +40,7 @@
 from packit.local_project import LocalProject
 
 from packit_service.config import ServiceConfig, GithubPackageConfigGetter
+from packit_service.constants import GITHUB_FAILURE_STATUS
```

Comment:
Please, use the `CommitStatus` enum instead:

https://github.com/packit-service/ogr/blob/63774270a240f9bd7064d33259417e7564a1e1a4/ogr/abstract.py#L536-L542

Reply:
I will refactor it

Reply:
Thanks!